Source/Rows/Common/FieldRow.swift

```diff
@@ -86,8 +86,25 @@ open class FieldRow<Cell: CellType>: FormatteableRow<Cell>, FieldRowConformance,
     open var keyboardReturnType: KeyboardReturnTypeConfiguration?
 
     /// The percentage of the cell that should be occupied by the textField
-    open var textFieldPercentage: CGFloat?
-
+	@available (*, deprecated, message: "Use titleLabelPercentage instead")
+	open var textFieldPercentage : CGFloat? {
+		get {
+			guard let titlePercentage = titlePercentage else { return nil }
+			return 1 - titlePercentage
```

Comment:
Look at https://www.natashatherobot.com/swift-using-map-to-deal-with-optionals/ that shows how to use map with optionals.